### ISSUE TITLE
[Feature Request] DNG conferter for Linux

### ISSUE THREAD

**Issue by nerdpaolino** [mirsadm/motioncam#147]
Is it possible to have the DNG converter also for Linux machine?
Many thanks.

**Comment by mirsadm** [comment]
Yes it is possible to compile it to run on Linux and I do intend on doing that. I'm already stretched for time supporting three platforms though. The good news is someone has reported that they work well under Wine so if you don't want to wait give that a go.

**Comment by nerdpaolino** [comment]
I need to wait because I wasn't be able to run it under wine, thanks.

**Comment by darkblaze69** [comment]
> 

Runs perfectly under wine. Just install vcrun2017, e.g winetricks vcrun2017.

**Comment by nerdpaolino** [comment]
I'm installing it now. Thanks for your help.

**Comment by nerdpaolino** [comment]
I've installed with winetricks vcrun2017, but when i launch wine motioncam-tools.exe it complains that it doesn't find a lot of dll's.
